common/lib/container-definitions/api-report/container-definitions.api.md

```diff
@@ -201,7 +199,7 @@ export interface IContainerContext extends IDisposable {
     // (undocumented)
     readonly submitSignalFn: (contents: any) => void;
     // (undocumented)
-    readonly taggedLogger?: ITelemetryBaseLogger;
+    readonly taggedLogger: ITelemetryBaseLogger;
```

Comment:
Seems like this is a breaking change, too. Maybe mention that taggedLogger is now required in the same place where the logger change is?